glitter/pages/admin.py

```diff
@@ -97,6 +82,31 @@ def wrapper(*args, **kwargs):
         ] + urlpatterns
         return urlpatterns
 
+    def get_inline_instances(self, request, obj=None):
+        if (
+                getattr(settings, 'GLITTER_PAGES_REMINDER', False) and
```

Comment:
Suggestion - instead of a setting, could we use the django apps framework?

eg.

```python
if apps.is_installed('glitter.reminders'):
    # ...
```